redash/tasks/queries.py

```diff
@@ -483,6 +512,7 @@ def refresh_schemas():
             logger.info(u"task=refresh_schema state=skip ds_id=%s reason=org_disabled", ds.id)
         else:
             refresh_schema.apply_async(args=(ds.id,), queue=settings.SCHEMAS_REFRESH_QUEUE)
+            get_all_table_samples.apply_async(args=(ds.id,), queue=settings.SCHEMAS_REFRESH_QUEUE)
```

Comment:
I'm wondering if this should an own Celery queue (and fallback to the schema refresh queue)?